lib/netsuite/records/employee.rb

```diff
@@ -1,20 +1,23 @@
 module NetSuite
   module Records
-
     class Employee < Support::Base
+      include Support::Fields
```

Comment:
`Fields` is already included in `Support::Base`

https://github.com/NetSweet/netsuite/blob/b0d659067335fc296ad9ab96f2e5458d4dfeae43/lib/netsuite/support/base.rb

Was there a reason you needed to include `Support::Fields` explicitly in this model?